### PR TITLE
Support pickles with tails

### PR DESCRIPTION
**Pull request by lvh**
A tail is stuff after the STOP instruction. This is hard to parse because genops() itself will raise.

### CODE REVIEW

File: src/pikara/analysis.py [hunk 1]
```diff
@@ -224,21 +226,25 @@ def _parse(pickle):
...
+    issues = []
     stack = []
     markstack = []
     stackslice = None
     memo = {}
     maxproto = -1
     op = arg = pos = None
 
-    def _raise(E, msg, **kwargs):
+    def _complain(E, msg, **kwargs):
```
Comment (Ciemaar) [inline src/pikara/analysis.py]: I would expect this to be called _may_raise or _collect_issue (with the functionality as written)

File: src/pikara/analysis.py [hunk 2]
```diff
@@ -224,21 +226,25 @@ def _parse(pickle):
...
         """
         Tiny helper for raising exceptions with lots of context.
         """
         entry = _ParseEntry(op=op, arg=arg, pos=pos, stackslice=stackslice)
```
Comment (Ciemaar) [inline src/pikara/analysis.py]: If we're including stackslice here we should clear it at the start of the loop to prevent including a stackslice from the prior instruction.

Comment (lvh) [reply src/pikara/analysis.py]: fixed
